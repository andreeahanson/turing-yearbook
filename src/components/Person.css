article {
  background-color: rgba(255, 255, 255, 0.95);
  min-width: 300px;
  max-width: 300px;
  min-height: 450px;
  margin: 20px; 
  border-radius: 7%;
}

img {
  height: 260px;
  width: 260px;
  border-radius: 7%;
  margin: 10px;
} 

.name {
  color: red;
}

.quote {
  font-style: italic;
}

.superlative {
  color: lightskyblue;
}
