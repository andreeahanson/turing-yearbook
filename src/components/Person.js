import React from 'react';
import './Person.css';

const Person = props => {
  return <article key={props.person.id}>
    <img src={props.person.photo}/>
    <h4 className="name">{props.person.name}</h4>
    <p className="quote">{props.person.quote}</p>
    <h4 className="superlative">{props.person.superlative}</h4>
  </article>

}

export default Person;
