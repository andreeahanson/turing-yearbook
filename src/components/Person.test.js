import React from 'react';
import { shallow, mount } from 'enzyme';
import Person from './Person.js';


describe('Person', () => {
  it('should match the snapshot with all data passed in correctly', () => {
    const wrapper = shallow(<Person 
    id={1}
    name = "Eric"
    quote = "You miss all the shots you don't take."
    superlative = "Most Likely to Win the PGA"
    photo = "./eric.jpg"
    />);

    expect(wrapper).toMatchSnapshot();
  })


})


