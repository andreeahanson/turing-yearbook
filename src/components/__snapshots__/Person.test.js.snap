// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Person should match the snapshot with all data passed in correctly 1`] = `
<article
  key="1"
>
  <img
    alt="animal"
    src="./eric.jpg"
  />
  <h4
    className="name"
  >
    Eric
  </h4>
  <p
    className="quote"
  >
    You miss all the shots you don't take.
  </p>
  <h4
    className="superlative"
  >
    Most Likely to Win the PGA
  </h4>
</article>
`;
